### ISSUE TITLE
[Bug] The back button simply closes the application within a task.

### ISSUE THREAD

**Issue by Terkyz** [Webierta/todo_manager#2]
Inside a task, if I use the method of my system (Android 11) to return to the main menu the application [suddenly closes](https://imgur.com/a/wXZZWDd), but if I press the upper left arrow to return to the main menu it works as desired.

The expected behavior, at least in my case, is that both my system method to go back and the upper left arrow do the same.

**Comment by Webierta** [comment]
Yes, initially that behavior of going back only with the arrow of the app's menu bar was what I have programmed, but surely you are right and I should change it.
